docs/markdown/Contributions/development/contributor-setup.md

```diff
@@ -26,6 +26,10 @@ We use the popular forking workflow typically used by open source projects. See
 Step 2: Bootstrap the Rust engine
 ---------------------------------
 
+Pants requires several dependencies to be installed: a Python 3.9 interpreter, Rust, the protobuf compiler, clang and others. The easiest and fastest way to set up an environment with all necessary dependencies is to use the Nix package manager. The Pants repository includes a suitable `shell.nix` file. Follow the instructions on the [Nix website](https://nixos.org/download.html) to install Nix. Then `cd` into the directory where you cloned the Pants repo and type `nix-shell`. This will download all the necessary dependencies and start a shell with a suitably configured PATH variable to make them available for use.
```

Comment:
I'd be wary of using wording that suggests this is a blessed way getting a dev environment up and running until we have reasonable quorum of contributors using it.

When we say "the easiest and fastest" way to do something is X, we're implicitly committing to supporting that methodology.  Since almost none of us have extensive experience with Nix,  I wouldn't want to claim or even suggest that.

More conservatively, we could claim there's experimental support for a Nix closure (is that the right terminology) that makes dev setup easy.

Reply:
Fair enough, I've changed the wording to reflect this.